Description: drop feature psa-crypto-conversions
 The feature psa-crypto-conversion requires a psa-crypto version higher than
 what we currently have in sid (0.9.2). We could package the latest psa-crypto
 (0.12 right now), but that in turn depends on mbedtls 3.0.0, which isn't in sid
 yet.

 Disable the feature psa-crypto-conversions for now.

Author: Emanuele Rocca <ema@debian.org>
Last-Update: 2023-10-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: cryptoki/Cargo.toml
===================================================================
--- cryptoki.orig/Cargo.toml
+++ cryptoki/Cargo.toml
@@ -66,5 +66,4 @@ version = "2.0"
 
 [features]
 generate-bindings = ["cryptoki-sys/generate-bindings"]
-psa-crypto-conversions = ["psa-crypto"]
 serde = ["secrecy/serde"]
